collector_statusc.js

```diff
@@ -31,7 +31,9 @@ class CollectorStatusC extends AlServiceC {
             },
             body: data
         };
-        return this.put(`/statuses/${statusId}/streams/${stream}`, payload);
+        // Few collector stream contain `/` which not excepted by collectors_status service. so Encode the stream before making the api call.It will encodes special characters including: , / ? : @ & = + $ # 
+        const encodedStream = encodeURIComponent(stream);
```

Comment:
could we maybe replace `/` characters with `__` (double underscore)? I'd like to keep the stream human readable if possible - URI encoding is hard for brains to parse

Reply:
discussed offline -- stream name is decoded on backend, so no need to change this

Reply:
Collectors_status service will decode the stream at their end. Instead encoding if we replace the with `__` then ,we have to clear the old status for googlestack driver.